### PR TITLE
wasm-client & mobile: bump taproot-assets to `v0.3.0-alpha`, litd to `v0.11.0-alpha` & loop to `v0.26.4-beta`

### PR DESCRIPTION
**Pull request by ViktorTigerstrom**
This PR bumps the wasm-client's taproot-assets dependency to `v0.3.0-alpha`, litd to `v0.11.0-alpha` & loop to `v0.26.4-beta`.

The reason why we do this PR before a litd release, is that we'd like to do a litd release that uses an LNC dependency with tapd `v0.3.0-alpha`, to update the gRPC stubs of tapd to v0.3.0. We'll then do a new litd release, and make one more PR for LNC that updates litd to the new release.

### CODE REVIEW

File: cmd/wasm-client/go.mod [hunk 1]
```diff
@@ -5,12 +5,12 @@ require (
...
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/lightninglabs/faraday v0.2.11-alpha
-	github.com/lightninglabs/lightning-node-connect v0.2.4-alpha
+	github.com/lightninglabs/lightning-node-connect v0.2.5-alpha
 	github.com/lightninglabs/lightning-terminal v0.10.5-alpha
 	github.com/lightninglabs/loop v0.26.2-beta
 	github.com/lightninglabs/pool v0.6.4-beta
-	github.com/lightninglabs/taproot-assets v0.2.3
-	github.com/lightningnetwork/lnd v0.16.4-beta
+	github.com/lightninglabs/taproot-assets v0.3.0
```
Comment (guggero) [inline cmd/wasm-client/go.mod:12]: Can we bump Loop to `v0.26.4-beta` and Litd to `v0.11.0` please?

**Review by guggero — COMMENTED** [review]
Just one request, other than that looks good to me!

File: cmd/wasm-client/go.mod (hunk 1 above)
Comment (ViktorTigerstrom) [reply cmd/wasm-client/go.mod:12]: Ok, will do!

**Comment by ViktorTigerstrom** [comment]
Also updated litd+loop. This PR will now unfortunately clash a bit with @itsrachelfish work, as I know she's working on changes that also does that. 

`itests` failed in the last build, though that was due to a timeout. They're passing this time!

**Review by jamaljsr — APPROVED** [review]
tACK LGTM 🚀

Tested via `make example-server`. No issues found.

**Pull request merged**